tests/cli/demultiplex/test_finish_demux.py

```diff
@@ -32,28 +32,6 @@ def test_finish_all_cmd_dry_run(
     assert result.exit_code == EXIT_SUCCESS
 
 
-def test_finish_all_cmd(
```

Comment:
Test does not do anything except log this: `"Flow cell is already finished!"`

Reply:
Yes and this test is broken anyway. If you debug it actually excepts a flow cell error in the old flow. Instead of raising it, it returns